### ISSUE TITLE
Incorrect error: ‘Private item used outside scope’ for public `lazy_static!` value

### ISSUE THREAD

**Issue by sanmai-NL** [intellij-rust/intellij-rust#1995]
The last nightly and presumably earlier versions exhibit a bug.

crate A:
```rust
lazy_static! {
    pub static ref BLA: () = {};
}
```

crate B:
```rust
use crate_a::BLA; \\ Error highlighted `A private item was used outside its scope. [E0603]`
```

The compiler does not complain, however.

**Comment by Undin** [comment]
Fixed in a0d2386
